dependencies.yaml

```diff
@@ -76,18 +88,30 @@ dependencies:
           - spdlog>=1.12.0,<1.13
   build_python:
     common:
+      - output_types: [conda]
+        packages:
+          - &rmm_conda rmm==24.4.*
```

Comment:
This should be in a list `depends_on_rmm`. I refactored this and a few other things in https://github.com/rapidsai/ucxx/pull/171/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6 but I'd like this PR to go in first, and reduce the diff of #171. Maybe just compare the two dependencies files and port over the changes you like into this PR?

Reply:
I backported this in b0142c2.